index.html

```diff
@@ -0,0 +1,475 @@
+                        <li class="header"><strong>OPINION</strong></li>
+                        <li><a>Today's Opinion</a></li>
+                        <li><a>Op-Ed Columnists</a></li>
+                        <li><a>Editorials</a></li>
+                        <li><a>Op-Ed Contributers</a></li>
+                        <li><a>Letters</a></li>
+                        <li><a>Sunday Review</a></li>
+                        <li><a>Video: Opinion</a></li>
+                    </ul>
+    
+                    <ul class = 'font-style'>
+                        <li class="header"><strong>ARTS</strong></li>
+                        <li><a>Today's Arts</a></li>
+                        <li><a>Art & Design</a></li>
+                        <li><a>Books</a></li>
+                        <li><a>Dance</a></li>
+                        <li><a>Movies</a></li>
+                        <li><a>Music</a></li>
+                        <li><a>Pop Culture</a></li>
+                        <li><a>Television</a></li>
+                        <li><a>Theatre</a></li>
+                        <li><a>Video: Arts</a></li>
+    
+                    </ul>
+    
+                    <ul class = 'font-style'> 
+                        <li class="header"><strong>LIVING</strong></li>
+                        <li><a>At Home</a></li>
+                        <li><a>Automobiles</a></li>
+                        <li><a>Games </a></li>
+                        <li><a>Education </a></li>
+                        <li><a>Food</a></li>
+                        <li><a>Health </a></li>
+                        <li><a>Jobs </a></li>
+                        <li><a>Love </a></li>
+                        <li><a>Magazine </a></li>
+                        <li><a>Parenting </a></li>
+                        <li><a>Real Estate </a></li>
+                        <li><a>Recipies </a></li>
+                        <li><a>Styles </a></li>
+                        <li><a>T magizine </a></li>
+                        <li><a>Travel </a></li>
+    
+                
+                    </ul>
+    
+                    <ul class = 'font-style'>
+                        <li class="header"><strong>MORE</strong></li>
+                        <li><a>Read Center </a></li>
+                        <li><a>Wirecutter </a></li>
+                        <li><a>Live Events</a></li>
+                        <li><a>The Learning Network </a></li>
+                        <li><a>Tools & Services </a></li>
+                        <li><a>Multimedia </a></li>
+                        <li><a>Photography </a></li>
+                        <li><a>Video </a></li>
+                        <li><a>Newsletter </a></li>
+                        <li><a>TimesMachine </a></li>
+                        <li><a>NYT Store </a></li>
+                        <li><a>Times Journey </a></li>
+                        <li><a>Manage My Account </a></li>
+    
+                    </ul>
+             </div>
+            </div>
```

Comment:
- [ ]  The list of categories at the bottom should be links, but the way you implement the anchor tag is wrong (without href). Please apply anchor tags properly for them.

Reply:
Thank you, Safa. Your review was insightful. 

To the next reviewer, please find that I have:
* added article tags where needed
* defined and used the float property as a CSS and HTML class (dfloat-l & dfloat-r)
* Updated the pull request name 
* Added a max-width of 300px to the left navbar icons section to avoid stretching on browsers other than Gzim 
 and my own.
* removed the node folder from the remote repo and added to .gitignore file.
* completed the href in the needed <a> tags